plugins/api-docs/src/components/ApiDefinitionCard/index.ts

```diff
@@ -14,4 +14,4 @@
  * limitations under the License.
  */
 
-export { ApiDefinitionCard } from './ApiDefinitionCard';
```

Comment:
> Index files that are re-exporting symbols from non-index files should always enumerate all exports, that is:

https://backstage.io/docs/architecture-decisions/adrs-adr004

I think we should keep it as-is?

Reply:
I agree, but I was getting the following error trying to re-export `type ApiDefinitionWidget` from `./ApiDefinitionCard`.

```
Re-exporting a type when the '--isolatedModules' flag is provided requires using 'export type'.ts(1205)
```

https://github.com/microsoft/TypeScript/issues/28481

It's the first time I'm running into this issue with TS that I recall. I tried separating the type into it's own file and changing it to an interface but no luck. It's possible this type doesn't need to be exported by the package, but will be required as a type for some of the other files/modules in the package. Anyone have recommendations on how to deal with this?

Reply:
Like the error message says, use `export type` :grin:

```ts
export { ApiDefinitionCard } from './ApiDefinitionCard';
export type { ApiDefinitionWidget } from './ApiDefinitionCard';
```

Reply:
Thanks! It was starring my right in the face. Odd that I've never run into this before.